src/puppetlabs/trapperkeeper/services/status/status_debug_logging.clj

```diff
@@ -0,0 +1,11 @@
+(ns puppetlabs.trapperkeeper.services.status.status-debug-logging
+  (:require [clojure.tools.logging :as log]
+            [schema.utils :refer [validation-error-explain]]
+            [cheshire.core :as json]
+            [puppetlabs.trapperkeeper.services.status.status-core :as status-core]))
+
+(defn log-status
+  "Log status information at the debug level as json"
+  []
+  (let [status (status-core/v1-status :debug)]
```

Comment:
I wonder if it would be worth adding a fn to core called, e.g., `status-latest-version` or `latest-version-status`, that would just call into `v1-status` for now.  Then, you could call `status-latest` from here, and if we ever introduce a `v2-status`, it would be fairly obvious that that new fn needed to be updated to call the newer version.  Otherwise a maintainer would need to know to come in here and change this line to the later version as well?

Or, maybe better, the core namespace could contain a fn like this:

```clj
(defn status-map-for-version [version level]
   (cond version
      nil? (v1-status level)
      ;; 2 (v2-status level ;; future
      1 (v1-status level)))
```

wdyt?